### PR TITLE
[4.0]remove unused method disconnectedCallback

### PR DESCRIPTION
**Pull request by bahl24**
Signed-off-by: Nitish Bahl <nitishbahl24@gmail.com>

Pull Request for Issue #23792  .

### Summary of Changes
Removed unused method disconnectedCallback

### Testing Instructions
* Build js files 
* Open the browser console
* Go to create a new article in the Joomla backend
* Reduce the browser window width until you see an error in the con

### Expected result
No error in console

### Actual result
Error in console

### CODE REVIEW

File: build/media_source/system/js/fields/joomla-field-user.w-c.es6.js [hunk 1]
```diff
@@ -59,11 +59,6 @@
       }
     }
 
-    disconnectedCallback() {
```
Comment (anuragteapot) [inline build/media_source/system/js/fields/joomla-field-user.w-c.es6.js:62]: I think removing the `disconnectedCallback` is not a solution.

Comment (bahl24) [reply build/media_source/system/js/fields/joomla-field-user.w-c.es6.js:62]: I could not find any usage/Reference of this method, in any form or modal.

Comment (anuragteapot) [reply build/media_source/system/js/fields/joomla-field-user.w-c.es6.js:62]: Check this https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements

**Comment by dgrammatiko** [comment]
@bahl24 please close this pr and head to MDN to read the specs

**Comment by dgrammatiko** [comment]
Also what you were supposed to fix with this was already patched here:
https://github.com/joomla/joomla-cms/pull/23881

**Comment by Quy** [comment]
@dgrammatiko Does your PR fix #23792?

**Comment by dgrammatiko** [comment]
Yes

**Pull request closed without merging**